doc/md/guides/clickhouse/getting-started.mdx

```diff
@@ -0,0 +1,536 @@
+<TabItem value="hcl" label="Atlas DDL (HCL)" default>
+
+```hcl title="schema.hcl"
+table "users" {
+  schema = schema.demo
+  engine = MergeTree
+  column "id" {
+    null = false
+    type = UInt64
+  }
+  column "name" {
+    null = false
+    type = String
+  }
+  primary_key {
+    columns = [column.id]
+  }
+}
+table "repos" {
+  schema = schema.demo
+  engine = MergeTree
+  column "id" {
+    type = UInt64
+    null = false
+  }
+  column "name" {
+    type = String
+    null = false
+  }
+// highlight-start
+  column "description" {
+    type = sql("Nullable(String)")
+    null = true
+  }
+// highlight-end
+  column "owner_id" {
+    type = Bigint
+    null = false
+  }
+  primary_key {
+    columns = [column.id]
+  }
+}
+// highlight-start
+table "commits" {
+  schema = schema.demo
+  engine = MergeTree
+  column "id" {
+    type = UInt64
+    null = false
+  }
+  column "message" {
+    type = String
+    null = false
+  }
+  column "repo_id" {
+    type = Bigint
+    null = false
+  }
+  column "author_id" {
+    type = Bigint
+    null = false
+  }
+  primary_key {
+    columns = [column.id]
+  }
+}
+// highlight-end
+schema "demo" {
+  engine  = sql("Memory")
+}
+
+```
+
+</TabItem>
+<TabItem value="sql" label="SQL">
+
+```sql title = "schema.sql"
+CREATE TABLE `users` (
+  `id` UInt64,
+  `name` String NOT NULL
+)  ENGINE = MergeTree
+   PRIMARY KEY (`id`)
+   SETTINGS index_granularity = 8192;
+
+CREATE TABLE `repos` (
+  `id` UInt64,
+  `name` String NOT NULL,
+// highlight-next-line
+  `description` String NULL,
+  `owner_id` Bigint NOT NULL
+) ENGINE = MergeTree
+  PRIMARY KEY (`id`)
+  SETTINGS index_granularity = 8192;;
+
+// highlight-start
+CREATE TABLE `commits` (
+  `id` UInt64,
+  `message` String NOT NULL,
+  `repo_id` Bigint NOT NULL,
+  `author_id` Bigint NOT NULL,
+)  ENGINE = MergeTree
+   PRIMARY KEY (`id`)
+   SETTINGS index_granularity = 8192;;
+// highlight-end
+```
+</TabItem>
+</Tabs>
+
+Next, let's run the `atlas migrate diff` command once more:
+
+```shell
+atlas migrate diff add_commits \
+  --to file://schema.hcl \
+  --dev-url "docker://clickhouse/23.11/dev"
+```
+
+Run `ls migrations`, and you'll notice that a new migration has been generated. Let's run `atlas migrate push` again and
+observe the new file on the migration directory page.
+
+```shell
+atlas migrate push app \
+  --dev-url "docker://clickhouse/23.11/dev"
+```
+
+<div style={{textAlign: 'center'}}>
+    <img src="https://atlasgo.io/uploads/clickhouse/images/second-push.png" alt="clickhouse migrate push" width="100%"/>
+    <p style={{fontSize: 12}}>Migration Directory created with <code>atlas migrate push</code></p>
+</div>
+
+## Next Steps
+
+In this guide we learned about the declarative and versioned workflows, and how to use Atlas to generate migrations,
+push them to an Atlas workspace and apply them to databases.
+
+Next, we recommend you [set up CI](/cloud/setup-ci) for your migration directory, to ensure that all changes applied to
```

Comment:
Next steps:
* Full docs for clickhouse HCL schema
* How to set up CI
* Deploying changes with Terraform, Kubernetes
* Learn about modern CI/CD for databases